setup.py

```diff
@@ -31,7 +31,7 @@ def get_version(rel_path: str) -> str:
     "click>=8.0.4",
     "httpx~=0.23.0",
     "GitPython>=3.1.29",
-    "rich[jupyter]~=13.1.0",
+    "rich~=13.1.0",
```

Comment:
So the regular rich package just works without modification?
Cool

Reply:
The regular rich still works for CLI. In Jupyter it's broken though until you install the [jupyter] optional dep.

We could check in the `install.py` script if it's being executed inside a jupyter env/if there is jupyter installed. But from what I've read this is not recommended, because e.g. if we were to ever build wheels, then the wheel doesn't execute the setup.py, and I think the result would be that `rich[jupyter]` will always be a dependency. At least that's what I read in some places. It is all surprisingly underdocumented and underexplained how it works

Reply:
BUT right now we could just put an `is_in_notebook()` check inside of setup.py that tries to `import IPython` and change the installed rich based on that